### PR TITLE
Revising couchbase tile

### PR DESCRIPTION
**Pull request by pohannigansfx**
Revamped to increase prescriptive guidance and polish text, but not using new jinja doc template because this tile draws partly from smart agent repo

### CODE REVIEW

File: couchbase/README.md [hunk 1]
```diff
@@ -46,25 +46,27 @@ Metadata associated with SignalFx's Couchbase integration can be found <a target
 
 ### USAGE
 
-Below are screen captures of dashboards created for this plugin by SignalFx, illustrating the metrics emitted by this plugin.
+Screen captures below of dashboards created for this plugin by SignalFx illustrate the metrics emitted by this plugin.
```
Comment (brianashby-sfx) [inline couchbase/README.md:49]: remove directional language. an alternative that i like using is "the following..."

File: couchbase/README.md [hunk 2]
```diff
@@ -46,25 +46,27 @@ Metadata associated with SignalFx's Couchbase integration can be found <a target
...
 
 **Note on bucket metrics**
 
-This plugin emits some metrics about the bucket's performance across the cluster, and some metrics about the bucket's performance per node.
+This plugin emits metrics about bucket performance across the cluster, and also metrics about bucket performance per node.
 
-Metrics beginning with `gauge.bucket.basic.​*` and `gauge.bucket.quota.*`​ are reported once per cluster. All other bucket metrics (`gauge.bucket.*`) are reported by every node that hosts that bucket. In order to analyze bucket performance for the entire bucket, apply functions like Sum or Mean to group node-level metrics together by bucket.
+Metrics beginning with `gauge.bucket.basic.​*` and `gauge.bucket.quota.*`​ are reported once per cluster. All other bucket metrics (`gauge.bucket.*`) are reported by every node that hosts that bucket.
+
+To analyze performance for the entire bucket, you can group node-level metrics together by applying functions like Sum or Mean to them.
```
Comment (brianashby-sfx) [inline couchbase/README.md:59]: Should "Sum" and "Mean" be in code format?

File: couchbase/README.md [hunk 3]
```diff
@@ -46,25 +46,27 @@ Metadata associated with SignalFx's Couchbase integration can be found <a target
...
 
-Below are screen captures of dashboards created for this plugin by SignalFx, illustrating the metrics emitted by this plugin.
+Screen captures below of dashboards created for this plugin by SignalFx illustrate the metrics emitted by this plugin.
 
 For general reference on how to monitor Couchbase, see <a target="_blank" href="http://blog.couchbase.com/monitoring-couchbase-cluster">Couchbase Monitoring</a> and <a target="_blank" href="http://developer.couchbase.com/documentation/server/4.0/monitoring/monitoring-rest.html">Monitor using the REST API</a>.
 
 **Note on bucket metrics**
 
-This plugin emits some metrics about the bucket's performance across the cluster, and some metrics about the bucket's performance per node.
+This plugin emits metrics about bucket performance across the cluster, and also metrics about bucket performance per node.
```
Comment (brianashby-sfx) [inline couchbase/README.md:55]: is emit the best word we can use here? maybe export, send, generate, create? if this is used across multiple tiles, better to keep consistent, though.

also, i think "plugin" should be "plug-in"

File: couchbase/README.md [hunk 4]
```diff
@@ -46,25 +46,27 @@ Metadata associated with SignalFx's Couchbase integration can be found <a target
...
 
-Metrics beginning with `gauge.bucket.basic.​*` and `gauge.bucket.quota.*`​ are reported once per cluster. All other bucket metrics (`gauge.bucket.*`) are reported by every node that hosts that bucket. In order to analyze bucket performance for the entire bucket, apply functions like Sum or Mean to group node-level metrics together by bucket.
+Metrics beginning with `gauge.bucket.basic.​*` and `gauge.bucket.quota.*`​ are reported once per cluster. All other bucket metrics (`gauge.bucket.*`) are reported by every node that hosts that bucket.
```
Comment (brianashby-sfx) [inline couchbase/README.md:57]: is the plug-in doing the reporting here?

File: couchbase/README.md [hunk 5]
```diff
@@ -46,25 +46,27 @@ Metadata associated with SignalFx's Couchbase integration can be found <a target
...
+
+To analyze performance for the entire bucket, you can group node-level metrics together by applying functions like Sum or Mean to them.
 
 **Monitoring a Couchbase cluster**
 
-On the Couchbase Nodes overview dashboard, you can see at a glance the status the nodes and buckets in a given cluster. Nodes in the cluster should be seeing balanced activity. Buckets in the cluster should each have adequate memory remaining.
+On the Couchbase Nodes overview dashboard, you can see at a glance the status of nodes and buckets in a given cluster. Nodes in the cluster should show balanced activity. Buckets in the cluster should each have adequate memory remaining.
```
Comment (brianashby-sfx) [inline couchbase/README.md:63]:
```suggestion
On the Couchbase Nodes overview dashboard, you can see the status of nodes and buckets in a given cluster. Nodes in the cluster should show balanced activity. Buckets in the cluster should each have adequate memory remaining.
```

Comment (brianashby-sfx) [reply couchbase/README.md:63]: what is "adequate memory" in this case?

File: couchbase/README.md [hunk 6]
```diff
@@ -46,25 +46,27 @@ Metadata associated with SignalFx's Couchbase integration can be found <a target
...
 
 ![Couchbase - Nodes and buckets in a cluster](././img/nodes_and_buckets_snapshot.png)
 
 *This cluster's three nodes have roughly the same number of gets per second, and its two buckets have plenty of headroom.*
 
-This dashboard also includes a percentile distribution of CPU utilization per node, allowing quick identification of unusually hot nodes. This chart shows minimum, 10th percentile, median (50th percentile), 90th percentile, and maximum CPU utilization for each node in the cluster.
+This dashboard also includes a percentile distribution of CPU utilization per node, enabling quick identification of unusually hot nodes. This chart shows minimum, 10th percentile, median (50th percentile), 90th percentile, and maximum CPU utilization for each node in the cluster.
```
Comment (brianashby-sfx) [inline couchbase/README.md:69]: is it obvious what a "hot node" is?

File: couchbase/README.md [hunk 7]
```diff
@@ -76,9 +78,9 @@ Zooming in to an individual node shows that node's activity, cache performance,
 
 ![Node overview](././img/node.png)
 
-*This node is lightly loaded. To compare its activity to other nodes in this cluster, we'd use the Couchbase Nodes dashboard above.*
+*This node is lightly loaded. To compare its activity to other nodes in this cluster, use the Couchbase Nodes dashboard above.*
```
Comment (brianashby-sfx) [inline couchbase/README.md:81]: is it obvious what "lightly loaded" means?

File: couchbase/README.md [hunk 8]
```diff
@@ -94,13 +96,13 @@ The Couchbase Buckets overview shows activity for all buckets being monitored.
 
 **Monitoring a single Couchbase bucket**
 
-Selecting a particular bucket to show on the Couchbase Bucket dashboard lets us go deep on that bucket's performance.
+Selecting a particular bucket to show on the Couchbase Bucket dashboard lets you monitor details of that bucket's performance.
```
Comment (brianashby-sfx) [inline couchbase/README.md:99]:
```suggestion
Selecting a particular bucket to show on the Couchbase Bucket dashboard enables you to monitor details of that bucket's performance.
```

File: couchbase/README.md (hunk 1 above)
Comment (pohannigansfx) [reply couchbase/README.md:49]: done

File: couchbase/README.md (hunk 4 above)
Comment (pohannigansfx) [reply couchbase/README.md:57]: The nodes are doing the reporting.

File: couchbase/README.md (hunk 3 above)
Comment (pohannigansfx) [reply couchbase/README.md:55]: Splunk usage dictionary lists "plugin" with no hyphen.

File: couchbase/README.md (hunk 6 above)
Comment (pohannigansfx) [reply couchbase/README.md:69]: It wasn't, but I added a few words (hot meaning > 80% utilization)

File: couchbase/README.md (hunk 5 above)
Comment (pohannigansfx) [reply couchbase/README.md]: Rewrote to remove ambiguity

**Comment by pohannigansfx** [comment]
Incorporated most of the review suggestions -- thank you, @brianashby-sfx

**Pull request merged**